docs/core/proto-docs.md

```diff
@@ -2882,7 +2882,7 @@ ReflectionService defines a service for application reflection.
 <a name="cosmos.base.snapshots.v1beta1.Metadata"></a>
 
 ### Metadata
-Metadata contains Cosmos SDK-specific snapshot metadata.
+Metadata contains SDK-specific snapshot metadata.
```

Comment:
If you're going to use just `SDK`, then it should be `an SDK ...`

Reply:
> @atheeshp can you remind me where I can read up on Amino AUX mode pls?

I don't think we have better docs for this yet, you may get some context here: #10249